### ISSUE TITLE
"dialog" is allowed as value of  "method" and "formmethod" attribute.

### ISSUE THREAD

**Issue by mtrootyy** [validator/validator#962]
```html
<!DOCTYPE html>
<html lang="">
<head>
<title>Test</title>
</head><body>
<dialog>
<form method="dialog">
<input type="submit" formmethod="dialog">
</form>
</dialog>
</body>
</html>
```
↓
```
    Warning: The dialog element is not supported in all browsers. Please be sure to test, and consider using a polyfill.
    From line 7, column 1; to line 7, column 8
    d>↩<body>↩<dialog>↩<form

    Error: Bad value dialog for attribute method on element form.
    From line 8, column 1; to line 8, column 22
    ↩<dialog>↩<form method="dialog">↩<inpu

    Error: Bad value dialog for attribute formmethod on element input.
    From line 9, column 1; to line 9, column 41
    "dialog">↩<input type="submit" formmethod="dialog">↩</for
```

`dialog` is allowed as value of `method` and `formmethod` attribute.
Ref:https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fs-method

Because `dialog` element is "Warning", `dialog` value of `method` and `formmethod` attribute also should be "Warning".

**Comment by mtrootyy** [comment]
Because dialog element is allowed(https://github.com/validator/validator/commit/d82970bafeb45d6b54eabad588ac200473570820), dialog value of method and formmethod attribute also should be allowed.

**Comment by sideshowbarker** [comment]
Fixed by https://github.com/validator/validator/pull/1443
